codecov.yml

```diff
@@ -7,15 +7,15 @@ codecov:
 coverage:
   precision: 2         # how many decimal places to display in the UI: 0 <= value <= 4
   round: down          # how coverage is rounded: down/up/nearest
-  range: 60...100      # custom range of coverage colors from red -> yellow -> green
+  range: 70...100      # custom range of coverage colors from red -> yellow -> green
```

Comment:
We are at 88% and have been since we started, so updating the range to more closely align with our current state.